notification/src/main/java/org/killbill/billing/notification/plugin/api/TagMetadata.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Groupon, Inc
```

Comment:
Wrong company :-) Check https://github.com/killbill/killbill-plugin-api/blob/d9eca5af0e37541069b1c608f95e100dbe13b301/.idea/copyright/apache.xml#L3 for the current Copyright profile.

Reply:
Hi @pierre 

Updated the copyright